src/Authorization/Controller/LoginController.php

```diff
@@ -50,11 +51,14 @@ final class LoginController extends AbstractApiController
     )]
     #[InvalidCredentialsResponse]
     #[DefaultResponses]
-    public function login(#[CurrentUser] User $user): JsonResponse
-    {
-        return $this->jsonResponse([
-            'username' => $user->getUserIdentifier(),
-            'roles' => $user->getRoles(),
-        ]);
+    public function login(
+        #[CurrentUser] User $user,
+        UserInformationHydratorInterface $userInformationHydrator
+    ): JsonResponse {
+        $userInformation = $userInformationHydrator->hydrate(
```

Comment:
What do you think if we would move this into the service? I guess we also should dispatch an event again if someone wants to add some additional data? Or did we decide that we will not do so for the user information?

Reply:
Good Point with the Event. At first i thought i will not introduce an Service since it will just call the Hydrator. 
But with the Event it totally make sense. 

I will add this.

Reply:
Also please keep in mind our blueprint, which states that the controller delegates to the service and the service then has the hydrator. :)

Reply:
Done :)